### ISSUE TITLE
denormalizationContext doesn't work while using output

### ISSUE THREAD

**Issue by sanhofman** [api-platform/api-platform#1876]
**API Platform version(s) affected**: 2.6.4

**Description**  
denormalizationContext doens't seem to work anymore while using output after updating to v2.6.4.

While using:

```/**
 * @ApiResource(
 *     collectionOperations={
 *         "post_test_api"={
 *              "path"= "/test",
 *              "method"="POST",
 *         }
 *     },
 *     itemOperations={},
 *     shortName="Test",
 *     normalizationContext={
 *          "groups"={"api-get"},
 *          "swagger_definition_name"="GET",
 *     },
 *     denormalizationContext={
 *          "groups"={"api-write"},
 *          "swagger_definition_name"="WRITE",
 *     },
 *     output=TestOutput::class
 * )
 */
class Test
```

The TestOuput-GET is always used in Swagger docs also while writing instead of Test-WRITE. This worked fine with v2.6.3.

Am I missing something?

**Comment by alanpoulain** [comment]
Duplicates of https://github.com/api-platform/core/issues/4212.

**Comment by alanpoulain** [comment]
Solved in 2.6.5.
